### PR TITLE
feat: add myt chunithm support

### PR DESCRIPTION
**Pull request by albshin**
Adds MYT Chunithm support. Class data is currently not supported, will be added in the future.

### CODE REVIEW

File: client/src/app/pages/dashboard/import/ImportPage.tsx [hunk 1]
```diff
@@ -212,15 +212,7 @@ function ImportInfoDisplayer({ game }: { game: Game }) {
...
-			<ImportInfoCard
-				name="MYT Exporter"
-				href="chunithm-myt-exporter"
-				desc="Export your scores from MYT."
-				moreInfo={
-					<>This is an userscript that exports scores from the MYT web interface.</>
-				}
-				key="MYT Exporter"
-			/>,
+			<ImportTypeInfoCard key="api/myt-chunithm" importType="api/myt-chunithm" />,
```
Comment (albshin) [inline client/src/app/pages/dashboard/import/ImportPage.tsx:215]: Replacing the user-script here with a proper API export

File: client/src/app/pages/dashboard/import/ImportPage.tsx [hunk 2]
```diff
@@ -547,6 +539,16 @@ function ImportTypeInfoCard({
...
+		case "api/myt-wacca":
+			return (
+				<ImportInfoCard
+					name="MYT Integration"
+					href="myt-wacca"
+					desc="Pull your WACCA scores from the MYT Network."
+					moreInfo="Note: All networks are reduced to their first three letters for anonymity reasons."
+					key="myt-wacca"
+				/>
+			);
```
Comment (albshin) [inline client/src/app/pages/dashboard/import/ImportPage.tsx:551]: Just doing some re-ordering in terms of alphabetical order

File: client/src/app/routes/ImportRoutes.tsx [hunk 3]
```diff
@@ -219,12 +219,15 @@ export default function ImportRoutes() {
...
-								<MytIntegrationPage game="wacca" />
+							<Route exact path="/import/myt-chunithm">
+								<MytIntegrationPage game="chunithm" />
 							</Route>
 							<Route exact path="/import/myt-ongeki">
 								<MytIntegrationPage game="ongeki" />
 							</Route>
+							<Route exact path="/import/myt-wacca">
+								<MytIntegrationPage game="wacca" />
+							</Route>
```
Comment (albshin) [inline client/src/app/routes/ImportRoutes.tsx:230]: More alphabetical sorting

**Comment by zkldi** [comment]
code all looks good, let me know when this is ready2merge

**Comment by albshin** [comment]
> code all looks good, let me know when this is ready2merge

This is ready to be merge

**Comment by zkldi** [comment]
```
✖⨯ Unable to compile TypeScript:
  src/lib/score-import/import-types/api/myt-chunithm/converter.test.ts(33,2):
  error TS2741: Property 'judgeHeaven' is missing in type '{ judgeCritical:
  number; judgeJustice: number; judgeAttack: number; judgeMiss: number;
  maxCombo: number; }' but required in type 'AsObject'.
```

**Pull request merged**